goblet/common_cloud_actions.py

```diff
@@ -11,17 +11,20 @@
     get_default_location,
     get_credentials,
 )
+from goblet.utils import get_python_runtime
 
 log = logging.getLogger("goblet.deployer")
 log.setLevel(logging.INFO)
 
 
-def create_cloudfunction(client, req_body, config=None):
+def create_cloudfunction(client: Client, params, config={}):
```

Comment:
instead of doing all the v1 and v2 checks, lets create seperate function create_cloudfunction and create_cloudfunction2 and we just do the check once and call it

Reply:
ok, that works. Or since it's only two places could we make the parent_key and operations 'calls' arguments that default to v1 behavior?